bundles/org.openhab.core.thing/src/test/java/org/openhab/core/thing/ThingUIDTest.java

```diff
@@ -26,19 +26,19 @@ public void testThreeSegments() {
         ThingTypeUID thingType = new ThingTypeUID("fake", "type");
         ThingUID t = new ThingUID(thingType, "gaga");
 
-        assertEquals("type", t.getThingTypeId());
+        assertEquals("type", t.getThingTypeUID().getId());
         assertEquals("gaga", t.getId());
         assertEquals("fake:type:gaga", t.getAsString());
     }
 
-    @Test
-    public void testTwoSegments() {
-        ThingUID t = new ThingUID("fake", "gaga");
-
-        assertNull(t.getThingTypeId());
-        assertEquals("gaga", t.getId());
-        assertEquals("fake::gaga", t.getAsString());
-    }
+    // @Test
+    // public void testTwoSegments() {
```

Comment:
Why was it commented?
Can it be uncommented, fixed or removed instead?

Reply:
I readed and changed it.